pinot-spi/src/main/java/org/apache/pinot/spi/utils/CommonConstants.java

```diff
@@ -500,6 +502,7 @@ public static class Request {
 
       public static class QueryOptionKey {
         public static final String TIMEOUT_MS = "timeoutMs";
+        public static final String EXTRA_PASSIVE_TIMEOUT_MS = "extraPassiveTimeoutMs";
```

Comment:
nit: shouldn't we use `extra` either in both the query option and the broker config or in neither (since they're configuring the same "passive" timeout value)?

Reply:
Yep. I'm renaming the broker config as pinot.broker.extraPassiveTimeoutMs as well